app/src/modules/files/routes/collection.vue

```diff
@@ -413,7 +421,7 @@ function useFileUpload() {
 							<v-button secondary @click="moveToDialogActive = false">
 								{{ t('cancel') }}
 							</v-button>
-							<v-button :loading="moving" @click="moveToFolder">
+							<v-button :loading="moving" :disabled="!batchEditAllowed" @click="moveToFolder">
```

Comment:
Let’s remove the redundancy here, as well as in the item.vue component.

```suggestion
							<v-button :loading="moving" @click="moveToFolder">
```